### ISSUE TITLE
Cannot load consento via expo on iOS

### ISSUE THREAD

**Issue by chmac** [consento-org/mobile#94]
**Describe the bug**

Edit: Updated to correct screenshot.

![image](https://user-images.githubusercontent.com/690997/91714739-9f5f6780-eb8c-11ea-9961-5e8ed061ea09.png)

**To Reproduce**

* Scan QR code from https://expo.io/@consento/consento
* Open in expo

**Expected behavior**

On [this page](https://consento.org/download.html) I see:

![image](https://user-images.githubusercontent.com/690997/91714355-d2552b80-eb8b-11ea-95be-7962668e7685.png)

So I expect to be able to open consento on expo in iOS.

**Comment by martinheidegger** [comment]
I can confirm this issue, it may be cause by an uncorrect publication of the ios app or that the ios app gets removed after 30 days of creation. Either way, valid bug. Thanks!

**Comment by chmac** [comment]
@martinheidegger Just shipped a [brand new demo app](https://expo.io/@chmac/generous-plans) to expo and it won't load on iOS. Same error. But it loads immediately on Android. Looks like this is an upstream expo issue and not specific to this project.

**Comment by martinheidegger** [comment]
Thank you for testing it out, and also thank you for posting on the expo forum page: https://forums.expo.io/t/expo-client-on-ios-is-unable-to-load-any-apps/42332

**Comment by martinheidegger** [comment]
Thank you for researching that. Seems like without team membership you wont be able to access consento through iOS like that. We need to update our documentation to highlight this.

**Comment by chmac** [comment]
@martinheidegger Yeah, looks like expo doesn't allow "public" access to apps on iOS, only on Android. Unfortunate, but sort of understandable, I guess Apple were not keen on the idea of shipping apps bypassing their review.

How will consento ship to iOS now? Are you already using TestFlight?

**Comment by chmac** [comment]
Another idea would be to create a "public" expo account that you could share multiple people could log into. Not sure how scalable that is, or what expo would say about it, but... :-)

**Comment by chmac** [comment]
@martinheidegger Just spotted [this](https://blog.expo.io/upcoming-limitations-to-ios-expo-client-8076d01aee1a) blog post. Apparently expo dropped the option to view "public" apps on their iOS client back in March 2018... :-(

**Comment by dkastl** [comment]
Somehow I remember now to have read the blog post in January after problems with iOS. But I don't remember what we did then. I think we just used the iOS app store and invited test users there.
